lib/sqeduler/service.rb

```diff
@@ -103,6 +103,8 @@ def config_redis_pool
           config.redis_pool
         else
           redis = { :namespace => "sqeduler" }.merge(config.redis_hash)
+          # Redis requires config hash to have symbols as keys.
+          redis = symbolize_keys(redis)
```

Comment:
I think I'd prefer to have this inlined on line 105 like so:

```
redis = { :namespace => "sqeduler" }.merge(symbolize_keys(config.redis_hash))
```

That way if `namespace` is provided as a string key, it will be properly merged in the resulting hash.